src/crawlers/cheerio_crawler.js

```diff
@@ -500,16 +522,15 @@ class CheerioCrawler {
     /**
      * Combines the provided `requestOptions` with mandatory (non-overridable) values.
      * @param {Request} request
-     * @param {Session?} session
      * @ignore
      */
-    _getRequestOptions(request, session) {
+    _getRequestOptions(request) {
```

Comment:
is removing `session` intended? there's a call to _getRequestOptions that uses the session parameter on line 481

Reply:
Done. That was a terrible mistake.